### PR TITLE
Create rentoshare-jdl.jdl

### PR DESCRIPTION
**Pull request by Mayesamomo**

### CODE REVIEW

File: rentoshare-jdl.jdl [hunk 1]
```diff
@@ -0,0 +1,137 @@
+
+application {
+  config {
+    baseName gateway // (1)
```
Comment (MathieuAA) [inline rentoshare-jdl.jdl]: why the comments and the numbers?

File: rentoshare-jdl.jdl [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    databaseType mongodb
+    devDatabaseType mongodb
+    prodDatabaseType mongodb
+    enableHibernateCache false
+    serverPort 8082
+    serviceDiscoveryType eureka
+  }
+  entities Property, Image
+}
+application {
```
Comment (MathieuAA) [inline rentoshare-jdl.jdl:57]: just for consistency, can you put an extra-line between l.52 and l.53 please?

Comment (MathieuAA) [reply rentoshare-jdl.jdl:57]: and between every `entity`, `relationship`, options, etc.

thanks

**Review by MathieuAA — CHANGES_REQUESTED** [review]

**Comment by MathieuAA** [comment]
The filename and the extension both contain "jdl", can you remove the one in the filename please? thanks.
I don't find the filename very clear, is it about sharing rentals?

File: rentoshare-jdl.jdl (hunk 1 above)
Comment (Mayesamomo) [reply rentoshare-jdl.jdl]: If you don't like anything on it , I'm sure you can change it yourself.

File: rentoshare-jdl.jdl (hunk 2 above)
Comment (Mayesamomo) [reply rentoshare-jdl.jdl:57]: Will do that.
Thanks

File: rentoshare-jdl.jdl [hunk 3]
```diff
@@ -0,0 +1,144 @@
+
+application {
+  config {
+    baseName gateway
+    reactive true 
+    packageName com.michael.developer.gateway
```
Comment (deepu105) [inline rentoshare-jdl.jdl:6]: Please remove the packagename so that default will be used

**Review by deepu105 — COMMENTED** [review]
Can you rename the file to be more descriptive plz

**Comment by mraible** [comment]
Closing due to lack of response.

**Pull request closed without merging**